lib/readyset/caches.rb

```diff
@@ -0,0 +1,19 @@
+module Readyset
+  class Caches
+    class << self
+      attr_reader :caches
+    end
+
+    def self.cache(id:, always: false)
```

Comment:
**_suggestion:_** Does `always` as a boolean parameter work better as a separate method?

Reply:
Hmm I'm not sure I follow -- can you give an example of what you mean?

This class defines a DSL where `.cache` is used to define caches to be used for cache migrations. `always` is a parameter for a given cache and indicates whether ReadySet will send queries to the upstream database if querying a cache fails for any reason. IMO, it's a _parameter_ of a cache as opposed to a separate entity, so it seemed appropriate to include it as a parameter of the `.cache` method

Reply:
I guess it depends on the caller of `.cache`.

 Like if it knows it is using `always` or not, and if that conditional logic should be surfaced more explicitly to reflect the behavior of what the boolean parameter represents.